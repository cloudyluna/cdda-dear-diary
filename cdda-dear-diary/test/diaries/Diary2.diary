Entry: 1/9, Year 1, Spring, day 61 08:02:25, 

Stats:
Strength: 14
Dexterity: 12
Intelligence: 12
Perception: 12
 
Skills:
computers: 1
electronics: 1
fabrication: 2
marksmanship: 2
mechanics: 1
survival: 2
devices: 1
health care: 2
athletics: 2
archery: 1
bashing weapons: 1
cutting weapons: 1
rifles: 3
piercing weapons: 1
vehicles: 3
social: 1
food handling: 1
tailoring: 1
applied science: 1

Proficiencies:
Lockpicking
Wound Care
Driving
Food Preparation
Garment Closures
Principles of Leatherworking
Pottery
Pottery Glazing

Mutations:
Open Perk Menu
Eye color: gray
Quick
Bookworm
Fast Reflexes
Sociable
Good Hearing
Outdoorsman
Masochist
Hair: brown, ponytail
Skin tone: light
Packmule
KÃ¼nstler
Fast Learner
Optimist
Pain Resistant
Night Vision
Pretty

Active missions:
Find some cosplay suits
Faction succession
 
THE WORLD HAS GONE NUTS.

LAWS ARE NO MORE.

I have to fend myself off from the monsters...and 
I think the humans are far scarier than those.

Who do I trust??


Wish me luck.



Entry: 2/9, Year 1, Spring, day 61 10:18:02, 2 hours, 15 minutes since last entry

Stats: 
Strength: 14 -> 25
 
Skills: 
marksmanship: 2 -> 3
survival: 2 -> 3
 
Gained Mutation: 
no skill rust
 
Kills: 
  16 Z zombies
   7 Z fat zombies
   1 Z decayed zombie
   5 z zombie children
   1 d Z-9
   1 Z tough zombie
   1 @ petrified person
   1 @ gawky guardian
 
Dear diary,

I just fought a swarm of zombies. Small and big. 
They were though but I managed.

I hope I can make it out.

I just need to fight.

I need to.

- Cat



Entry: 3/9, Year 1, Spring, day 61 12:10:30, 1 hour, 52 minutes since last entry

Lost Mutation: 
no skill rust
 
Kills: 
   2 Z zombies
 
Dear diary,

I found a little shack of some kind.

Along the way, I encountered a shopping cart. So 
my back will be relief of the loot burden.

I feel a bit better now. I hope this will last.

- Cat



Entry: 4/9, Year 1, Spring, day 61 13:11:20, 1 hour, 0 minutes since last entry

Dear diary,

I saw a group of beavers today.

I wish I would've give a dam, but no, so I moved 
on.

- Cat



Entry: 5/9, Year 1, Spring, day 61 14:44:37, 1 hour, 33 minutes since last entry

Skills: 
devices: 1 -> 2
 
Kills: 
   1 Z zombies
   1 Z zombie cop
   1 z zombie children
   1 @ feral human
 
Dear diary,

I was looting an abandoned motel.

Place doesn't seems too shabby, other than the two
swimming zombies.

Goodness, I think that was my first
encounter with a zombie dressed with hard-plates 
all over.

- Cat



Entry: 6/9, Year 1, Spring, day 61 15:45:42, 1 hour, 1 minute since last entry

Dear diary,

I've found a large compound of some kind.

Weird thing is..I didn't see any critters 
whatsoever.

I'll investigate.

- Cat



Entry: 7/9, Year 1, Spring, day 61 22:23:04, 6 hours, 37 minutes since last entry

Kills: 
   2 Z zombies
   1 y oversized moth fly
   4 v American crows
 
Dear diary,

that freaking moth WAS HUGE.

- Cat



Entry: 8/9, Year 1, Spring, day 62 06:50:05, 8 hours, 27 minutes since last entry

Skills: 
computers: 1 -> 0
electronics: 1 -> 0
mechanics: 1 -> 0
archery: 1 -> 0
applied science: 1 -> 0
 
New missions:
Find Relic
 
Kill Jabberwock
 
Find Inhaler
 
Find Antibiotics
 
Dear diary,

It was a logging facility with survivors.

They aren't really friendly though.

I gotta leave, quickly.

- Cat



Entry: 9/9, Year 1, Spring, day 62 08:32:23, 1 hour, 42 minutes since last entry

Skills: 
computers: 0 -> 1
 
New missions:
Reach Refugee Center
 
Kills: 
   1 Z zombies
   1 Z fat zombies
   2 Z zombie technicians
   2 @ deranged axemen
   4 Z zombie runners
   1 @ feral mechanic
   1 Z tough zombies
   1 v wild turkey
 
Dear diary,

I visited an evac shelter earlier.

It got overan by the zeds.

What the hell happened?

- Cat



