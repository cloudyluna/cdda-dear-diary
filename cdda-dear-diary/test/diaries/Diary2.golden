Right (MkDiary {totalIndexes = 9, entries = [MkEntry {index = 1, dateTime = MkDateTime {season = Spring, day = 61, year = 1, time = MkTime {hour = 8, minute = 2, second = 25}}, lastWrittenTime = Nothing, body = MkBody {stats = Just (Initiation (MkCharacterStats {strength = 14, dexterity = 12, intelligence = 12, perception = 12})), skills = Just (Initiation (MkCharacterSkills {computers = 1, electronics = 1, fabrication = 2, marksmanship = 2, mechanics = 1, survival = 2, devices = 1, healthCare = 2, athletics = 2, archery = 1, bashingWeapons = 1, cuttingWeapons = 1, rifles = 3, piercingWeapons = 1, vehicles = 3, social = 1, foodHandling = 1, tailoring = 1, appliedScience = 1})), proficiencies = Just (Initiation ["Lockpicking","Wound Care","Driving","Food Preparation","Garment Closures","Principles of Leatherworking","Pottery","Pottery Glazing"]), mutations = Just (Initiation ["Open Perk Menu","Eye color: gray","Quick","Bookworm","Fast Reflexes","Sociable","Good Hearing","Outdoorsman","Masochist","Hair: brown, ponytail","Skin tone: light","Packmule","K\252nstler","Fast Learner","Optimist","Pain Resistant","Night Vision","Pretty"]), activeMissions = Just (Initiation ["Find some cosplay suits","Faction succession"]), newMissions = Nothing, gainedMutation = Nothing, lostMutation = Nothing, kills = Nothing, npcKilled = Nothing, note = "THE WORLD HAS GONE NUTS.\n\nLAWS ARE NO MORE.\n\nI have to fend myself off from the monsters...and \nI think the humans are far scarier than those.\n\nWho do I trust??\n\n\nWish me luck.\n\n\n\n"}},MkEntry {index = 2, dateTime = MkDateTime {season = Spring, day = 61, year = 1, time = MkTime {hour = 10, minute = 18, second = 2}}, lastWrittenTime = Just (MkLastWrittenTime {day = Nothing, hour = Just 2, minute = 15}), body = MkBody {stats = Just (Update (fromList [("Strength",MkStatUpdate {from = 14, to = 25})])), skills = Just (Update (fromList [("marksmanship",MkStatUpdate {from = 2, to = 3}),("survival",MkStatUpdate {from = 2, to = 3})])), proficiencies = Nothing, mutations = Nothing, activeMissions = Nothing, newMissions = Nothing, gainedMutation = Just (Update ["no skill rust"]), lostMutation = Nothing, kills = Just (Update [MkCharacterKills {count = 16, symbol = "Z", name = "zombies"},MkCharacterKills {count = 7, symbol = "Z", name = "fat zombies"},MkCharacterKills {count = 1, symbol = "Z", name = "decayed zombie"},MkCharacterKills {count = 5, symbol = "z", name = "zombie children"},MkCharacterKills {count = 1, symbol = "d", name = "Z-9"},MkCharacterKills {count = 1, symbol = "Z", name = "tough zombie"},MkCharacterKills {count = 1, symbol = "@", name = "petrified person"},MkCharacterKills {count = 1, symbol = "@", name = "gawky guardian"}]), npcKilled = Nothing, note = "Dear diary,\n\nI just fought a swarm of zombies. Small and big. \nThey were though but I managed.\n\nI hope I can make it out.\n\nI just need to fight.\n\nI need to.\n\n- Cat\n\n\n\n"}},MkEntry {index = 3, dateTime = MkDateTime {season = Spring, day = 61, year = 1, time = MkTime {hour = 12, minute = 10, second = 30}}, lastWrittenTime = Just (MkLastWrittenTime {day = Nothing, hour = Just 1, minute = 52}), body = MkBody {stats = Nothing, skills = Nothing, proficiencies = Nothing, mutations = Nothing, activeMissions = Nothing, newMissions = Nothing, gainedMutation = Nothing, lostMutation = Just (Update ["no skill rust"]), kills = Just (Update [MkCharacterKills {count = 2, symbol = "Z", name = "zombies"}]), npcKilled = Nothing, note = "Dear diary,\n\nI found a little shack of some kind.\n\nAlong the way, I encountered a shopping cart. So \nmy back will be relief of the loot burden.\n\nI feel a bit better now. I hope this will last.\n\n- Cat\n\n\n\n"}},MkEntry {index = 4, dateTime = MkDateTime {season = Spring, day = 61, year = 1, time = MkTime {hour = 13, minute = 11, second = 20}}, lastWrittenTime = Just (MkLastWrittenTime {day = Nothing, hour = Just 1, minute = 0}), body = MkBody {stats = Nothing, skills = Nothing, proficiencies = Nothing, mutations = Nothing, activeMissions = Nothing, newMissions = Nothing, gainedMutation = Nothing, lostMutation = Nothing, kills = Nothing, npcKilled = Nothing, note = "Dear diary,\n\nI saw a group of beavers today.\n\nI wish I would've give a dam, but no, so I moved \non.\n\n- Cat\n\n\n\n"}},MkEntry {index = 5, dateTime = MkDateTime {season = Spring, day = 61, year = 1, time = MkTime {hour = 14, minute = 44, second = 37}}, lastWrittenTime = Just (MkLastWrittenTime {day = Nothing, hour = Just 1, minute = 33}), body = MkBody {stats = Nothing, skills = Just (Update (fromList [("devices",MkStatUpdate {from = 1, to = 2})])), proficiencies = Nothing, mutations = Nothing, activeMissions = Nothing, newMissions = Nothing, gainedMutation = Nothing, lostMutation = Nothing, kills = Just (Update [MkCharacterKills {count = 1, symbol = "Z", name = "zombies"},MkCharacterKills {count = 1, symbol = "Z", name = "zombie cop"},MkCharacterKills {count = 1, symbol = "z", name = "zombie children"},MkCharacterKills {count = 1, symbol = "@", name = "feral human"}]), npcKilled = Nothing, note = "Dear diary,\n\nI was looting an abandoned motel.\n\nPlace doesn't seems too shabby, other than the two\nswimming zombies.\n\nGoodness, I think that was my first\nencounter with a zombie dressed with hard-plates \nall over.\n\n- Cat\n\n\n\n"}},MkEntry {index = 6, dateTime = MkDateTime {season = Spring, day = 61, year = 1, time = MkTime {hour = 15, minute = 45, second = 42}}, lastWrittenTime = Just (MkLastWrittenTime {day = Nothing, hour = Just 1, minute = 1}), body = MkBody {stats = Nothing, skills = Nothing, proficiencies = Nothing, mutations = Nothing, activeMissions = Nothing, newMissions = Nothing, gainedMutation = Nothing, lostMutation = Nothing, kills = Nothing, npcKilled = Nothing, note = "Dear diary,\n\nI've found a large compound of some kind.\n\nWeird thing is..I didn't see any critters \nwhatsoever.\n\nI'll investigate.\n\n- Cat\n\n\n\n"}},MkEntry {index = 7, dateTime = MkDateTime {season = Spring, day = 61, year = 1, time = MkTime {hour = 22, minute = 23, second = 4}}, lastWrittenTime = Just (MkLastWrittenTime {day = Nothing, hour = Just 6, minute = 37}), body = MkBody {stats = Nothing, skills = Nothing, proficiencies = Nothing, mutations = Nothing, activeMissions = Nothing, newMissions = Nothing, gainedMutation = Nothing, lostMutation = Nothing, kills = Just (Update [MkCharacterKills {count = 2, symbol = "Z", name = "zombies"},MkCharacterKills {count = 1, symbol = "y", name = "oversized moth fly"},MkCharacterKills {count = 4, symbol = "v", name = "American crows"}]), npcKilled = Nothing, note = "Dear diary,\n\nthat freaking moth WAS HUGE.\n\n- Cat\n\n\n\n"}},MkEntry {index = 8, dateTime = MkDateTime {season = Spring, day = 62, year = 1, time = MkTime {hour = 6, minute = 50, second = 5}}, lastWrittenTime = Just (MkLastWrittenTime {day = Nothing, hour = Just 8, minute = 27}), body = MkBody {stats = Nothing, skills = Just (Update (fromList [("appliedScience",MkStatUpdate {from = 1, to = 0}),("archery",MkStatUpdate {from = 1, to = 0}),("computers",MkStatUpdate {from = 1, to = 0}),("electronics",MkStatUpdate {from = 1, to = 0}),("mechanics",MkStatUpdate {from = 1, to = 0})])), proficiencies = Nothing, mutations = Nothing, activeMissions = Nothing, newMissions = Just (Update ["Find Relic","Kill Jabberwock","Find Inhaler","Find Antibiotics"]), gainedMutation = Nothing, lostMutation = Nothing, kills = Nothing, npcKilled = Nothing, note = "Dear diary,\n\nIt was a logging facility with survivors.\n\nThey aren't really friendly though.\n\nI gotta leave, quickly.\n\n- Cat\n\n\n\n"}},MkEntry {index = 9, dateTime = MkDateTime {season = Spring, day = 62, year = 1, time = MkTime {hour = 8, minute = 32, second = 23}}, lastWrittenTime = Just (MkLastWrittenTime {day = Nothing, hour = Just 1, minute = 42}), body = MkBody {stats = Nothing, skills = Just (Update (fromList [("computers",MkStatUpdate {from = 0, to = 1})])), proficiencies = Nothing, mutations = Nothing, activeMissions = Nothing, newMissions = Just (Update ["Reach Refugee Center"]), gainedMutation = Nothing, lostMutation = Nothing, kills = Just (Update [MkCharacterKills {count = 1, symbol = "Z", name = "zombies"},MkCharacterKills {count = 1, symbol = "Z", name = "fat zombies"},MkCharacterKills {count = 2, symbol = "Z", name = "zombie technicians"},MkCharacterKills {count = 2, symbol = "@", name = "deranged axemen"},MkCharacterKills {count = 4, symbol = "Z", name = "zombie runners"},MkCharacterKills {count = 1, symbol = "@", name = "feral mechanic"},MkCharacterKills {count = 1, symbol = "Z", name = "tough zombies"},MkCharacterKills {count = 1, symbol = "v", name = "wild turkey"}]), npcKilled = Nothing, note = "Dear diary,\n\nI visited an evac shelter earlier.\n\nIt got overan by the zeds.\n\nWhat the hell happened?\n\n- Cat\n\n\n\n"}}]})